concourse/scripts/pipecleaner.py

```diff
@@ -137,6 +164,18 @@ def check_pipeline(self, data):
                         for i in item['config']['outputs']:
                             output_resources.add(i['name'])
 
+                    if 'run' in item['config']:
+                        if item['config']['run']['path'] in ['sh', 'bash', 'dash', 'ksh']:
```

Comment:
my point only was that if we call 'ash' in the pipeline instead of sh then the script won't get checked. maybe we should have some other test to ensure we only use 'sh' and 'bash'...